front/src/components/boxs/weather/WeatherBox.jsx

```diff
@@ -124,69 +126,78 @@ const WeatherBox = ({ children, ...props }) => (
             </div>
           </div>
           <div
-            class="col-6 text-right"
+            class="col-3 text-right"
             style={{
-              padding: '10px'
+              paddingRight: '10px',
+              paddingLeft: '10px',
+              marginTop: '-0.75rem'
             }}
           >
-            {props.weather === 'rain' && (
-              <i
-                class="fe fe-cloud-rain"
-                style={{
-                  fontSize: '60px'
-                }}
-              />
-            )}
-            {props.weather === 'clear' && (
-              <i
-                class="fe fe-sun"
-                style={{
-                  fontSize: '60px'
-                }}
-              />
-            )}
-            {props.weather === 'cloud' && (
-              <i
-                class="fe fe-cloud"
-                style={{
-                  fontSize: '60px'
-                }}
-              />
-            )}
-            {props.weather === 'snow' && (
+            <i
+              className={'fe ' + props.weather_icon}
+              style={{
+                fontSize: '50px'
+              }}
+            />
+          </div>
+        </div>
+        {props.display_mode[GetWeatherModes.AdvancedWeather] && (
+          <div className="col-9" style={{ padding: '0' }}>
+            <span>
               <i
-                class="fe fe-cloud-snow"
+                class="fe fe-droplet"
                 style={{
-                  fontSize: '60px'
+                  fpaddingRight: '5px'
                 }}
               />
-            )}
-            {props.weather === 'fog' && (
-              <i
-                class="fe fe-cloud"
+              {props.humidity}
+              <span
                 style={{
-                  fontSize: '60px'
+                  fontSize: '12px',
+                  color: 'grey'
                 }}
-              />
-            )}
-            {props.weather === 'drizzle' && (
+              >
+                <Text id="global.percent" />
+              </span>
+            </span>
+            <span style={{ float: 'right' }}>
```

Comment:
We may prefer bootstrap class usage instead of explicit styling: `class="float-right"`.

Reply:
+1

Reply:
Fixed